doc/configuration.rst

```diff
@@ -598,6 +598,35 @@ examples into a single mini-gallery, e.g.:
 For such a mini-gallery, specifying a custom heading message is recommended
 because the default message is vague: "Examples of one of multiple objects".
 
+Create mini-galleries using filepaths
+"""""""""""""""""""""""""""""""""""""
+Sometimes you may want to explicitly create a mini-gallery for files that may be 
+in the same location but not have functions in common, for example a set of 
+tutorials. The mini-gallery directive supports passing in a string path and 
+a glob-style string to a set of files. 
+
+.. code-block:: rst
+
+    .. minigallery:: numpy.exp ../examples/plot_0_sin.py ../examples/plot_4*
+
+
+.. minigallery:: numpy.exp ../examples/plot_0_sin.py ../examples/plot_4*
+   :add-heading: Mini-gallery using paths
+   :heading-level: -
+
+The order of the gallery entries is determined by the argument order. The ordering of 
+files matching a specific glob pattern, e.g. ``../examples/plot_4*`` can be set using the 
+``minigallery_sort_order`` configuration key. This configuration key supports
+any callable that is a valid *key* argument to ``sorted``::
+
+    sphinx_gallery_conf = {
+    ...,
+    # put duplicate figure at end 
+    'minigallery_sort_order' : lambda path: not path.startswith('plot_4b')
```

Comment:
Wouldn't this be better, i.e., sort first based on "plot_4b"-ness then on the filename itself?
```suggestion
    'minigallery_sort_order' : lambda path: (not path.startswith('plot_4b'), path)
```